packages/space-opera/editor/index.html

```diff
@@ -86,50 +103,57 @@
   }
 
   me-tabs {
-    background-color: #202124;  /* GOOGLE_GREY_900 */
+    background-color: var(--expandable-section-background);
     padding: 0;
-    width: 360px;
+    width: 440px;
+    border-left: 1px solid var(--tabs-conatiner-border)
   }
 
   me-tabs a {
     color: #dadcff; /* GOOGLE_GREY_300, but.. bluer */
   }
 
   me-tabbed-panel {
-    max-height: calc(100vh - 96px);
+    max-height: calc(100vh - 45px);
     overflow-y: auto;
     display: block;
   }
 </style>
 <body>
-  <model-editor>
-    <div class="app">
+<model-editor>
+  <div class="app">
     <div class="editor-body-root">
       <div class="mvContainer">
         <model-viewer-preview id="editing_adapter">
         </model-viewer-preview>
       </div>
       <me-tabs>
-        <me-tabbed-panel icon="save">
-          <model-viewer-snippet></model-viewer-snippet>
+        <me-tabbed-panel icon="home">
+          <home-container></home-container>
+        </me-tabbed-panel>
+        <me-tabbed-panel icon="import_export">
+          <me-export-panel></me-export-panel>
         </me-tabbed-panel>
         <me-tabbed-panel icon="create">
+          <me-export-panel header="true"></me-export-panel>
           <me-transformation-controls></me-transformation-controls>
           <me-ibl-selector></me-ibl-selector>
           <me-animation-controls></me-animation-controls>
           <me-hotspot-panel></me-hotspot-panel>
           <me-poster-controls></me-poster-controls>
         </me-tabbed-panel>
-        <me-tabbed-panel icon="color_lens">
-          <me-materials-panel></me-materials-panel>
-        </me-tabbed-panel>
         <me-tabbed-panel icon="photo_camera">
+          <me-export-panel header="true"></me-export-panel>
           <me-camera-settings></me-camera-settings>
         </me-tabbed-panel>
+        <me-tabbed-panel icon="color_lens">
+          <me-materials-panel></me-materials-panel>
+        </me-tabbed-panel>
```

Comment:
The inspector panel is slightly wider than the others and causes the preview to resize.

Reply:
Doesn't seem to repro for the astronaut; I'm guessing if the JSON lines are too long it wraps wrong or something?

Reply:
yep, fixed